_posts/blog/2016-09-27-droidcon_nyc.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "Droidcon NYC 2016"
+description: Deep knowledge from the best Android developers in the world.
+modified:
+categories: blog
+author: florina_muntenescu
+excerpt: Deep knowledge was shared by some of the best Android developers in the world at Droidcon NYC 2016. Here are some of the things we learned.
+tags: [Conference, Architecture, MVC, MVP, MVVM, RecyclerView, ConstraintLayout]
+date: 2016-09-27T10:39:55-04:00
+---
+upday was present at Droidcon NYC were we had a talk on <a href="http://www.slideshare.net/FlorinaMuntenescu/a-journey-through-mv-wonderland-66570666">MV* architecture</a>, a topic we also started on the blog. While there, but also via the live stream, we got the chance to see more than sixty top developers from some of the most renowned companies, sharing their knowledge on the most interesting topics in Android today. Talks on subjects like performance, Material Design, architecture, deep understanding of different types of Views or Layouts and war stories were all gathered at Droidcon NYC in two intense days. Here are some of the talks, tips or libraries that we considered the most interesting. Until the videos are posted, we're providing the links to the slides, where available.
```

Comment:
Well, only one of us gave the talk. I think it's fine to write this from a personal perspective. It's up to you, but if it were me, I'd say something like:
> I've just got back from DroidCon NYC..... were I gave my talk on ...... . While I was there....